apps/admin_app/lib/admin_app/honeydew_workers/export_data_worker.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Avia.ExportDataWorker do
+  @behaviour Honeydew.Worker
+
+  alias Snitch.Core.Tools.MultiTenancy.Repo
+  alias AdminAppWeb.Exporter
+
+  def export_data(%{"tenant" => tenant, "format" => format, "type" => type, "user" => user}) do
+    Repo.set_tenant(tenant)
+
+    case format do
+      "csv" ->
+        Exporter.csv_exporter(user, type)
+
+      "xlsx" ->
+        Exporter.xlsx_exporter(user, type)
+    end
+  end
```

Comment:
@ashish173  @SagarKarwande @arjun289 @jyotigautam @anantanant2015 @vikramjadon921995 @ramansah 
What might be wrong in this approach?
I think its related to how atoms behave in elixir and they are limited.

```elixir
  def export_data(%{"tenant" => tenant, "format" => format, "type" => type, "user" => user}) do
    Repo.set_tenant(tenant)
    apply(Exporter, :"#{format}_exporter", [user, type])
  end
```

`RAILS`
- in rails we do `send` with string params. There it should not be an issue. since not using symbols.